src/client/java/daos-java/src/main/native/DunsAttribute.pb-c.c

```diff
@@ -666,20 +678,34 @@ static const ProtobufCFieldDescriptor uns__duns_attribute__field_descriptors[7]
     0,             /* flags */
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
+  {
+    "no_prefix",
+    9,
+    PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_BOOL,
+    0,   /* quantifier_offset */
+    offsetof(Uns__DunsAttribute, no_prefix),
+    NULL,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)